woocommerce-gateway-paypal-express-checkout.php

```diff
@@ -29,6 +29,9 @@
 
 define( 'WC_GATEWAY_PPEC_VERSION', '2.1.1' );
 
+// Define plugin base
+define( 'WC_GATEWAY_PPEC_PLUGIN_BASE', plugin_basename( __FILE__ ) );
```

Comment:
Nothing against this in particular, but we're introducing a new global that is only used in one place.
Couldn't we just "assume" the path is `woocommerce-gateway-paypal-express-checkout/woocommerce-gateway-paypal-express-checkout.php` above?
This only seems useful if someone has renamed the plugin, but that seems unlikely and not necessarily something we should support. Thoughts?

Is there a particular reason you want to define a new global with the file name? Couldn't we

Reply:
I added it for 2 reasons.

1. It's one of the recommended methods to get the plugin path.
2. Should work even of the plugin folder is renamed, which can happen.

> This only seems useful if someone has renamed the plugin, but that seems unlikely and not necessarily something we should support

Given that we have a way to support even if the plugin folder is renamed, I think we should keep it.

Reply:
Fair enough. My point was that we don't do the same to detect PayPal Payments or something so maybe it shouldn't be _that_ necessary. Having said that, there's a `$file` public var in `WC_Gateway_PPEC_Plugin` that we could use instead of adding a new global. Could you make the change to use that?

Reply:
I totally missed that. `$file` works. I've added a commit for the same.

Reply:
Changes updated